### PR TITLE
Added ShowInterfacesCapabities, fixed ShowRunInterface, fixed ShowSwi…

### PR DESCRIPTION
**Pull request by smitsialis**
…tchStackPortSummary

## Description
- Added Show Interfaces Capabilities
- Fixed Show Switch StackPort Summary to support interfaces with the slash notation
- Modified Show Run interfaces to include speed and duplex parsing

## Motivation and Context
- Show Interfaces Capabilities is a new parser
- Show Switch StackPort summary was only working for certain models.   This now works for other models that have a slash in the interface name
- Added missing speed and duplex parsing to Show run interfaces

## Impact (If any)
None

## Screenshots:
image

image

image

## Checklist:

- [ x ] I have updated the changelog.
- [  ] I have updated the documentation (If applicable).
- [ x ] I have added tests to cover my changes (If applicable).
- [ x ] All new and existing tests passed.
- [ x  ] All new code passed compilation.

### CODE REVIEW

**Comment by ademz** [comment]
Hello @smitsialis ,

Thank you for your contribution. Can you please merge the latest version and fix the conflicts? 

Adem Oztas

**Comment by stevedhn** [comment]
Hi pls resolve the conflicts by merging with the lastest dev branch

File: changelog/undistributed/changelog_multiple.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+--------------------------------------------------------------------------------
+                            New
+--------------------------------------------------------------------------------
+* IOSXE,IOS
+    * Added ShowInterfacesCapabilities:
+        * show interfaces {interface} capabilities
```
Comment (GerriorL) [inline changelog/undistributed/changelog_multiple.rst:7]: Please separate IOS and IOSXE

**Review by GerriorL — COMMENTED** [review]
Code looks good, just one small change to the changelog

**Review by GerriorL — CHANGES_REQUESTED** [review]
Unfortunately we can't make schema changes as we need to preserve backwards compatibility

**Pull request closed without merging**